charts/camunda-platform/templates/ingress.yaml

```diff
@@ -30,7 +30,7 @@ spec:
               service:
```

Comment:
This was on purpose because running Web Modeler on a context path (like www.example.com/modeler) is not yet supported. We're planning to implement this feature until the 8.2 release though.

Reply:
I'd suggest adding it anyway as the rest of its config, where it's disabled by default and it should not appear anywhere (unless explicitly enabled)

Reply:
Hm, does it actually make sense to add it already? Users might think that they can also configure a `contextPath` for Web Modeler (similar to the other applications), but it wouldn't work anyway since it's not supported by the application itself.
It might be a good idea though to add a note to the README file about the missing support for Web Modeler if you'd like to run all apps on different paths of a single domain.
What do you think? :)

Reply:
We agreed to not add Web Modeler to the global Ingress yet. It will be added later though once it is supported to run Web Modeler on a context path (see https://github.com/camunda/web-modeler/issues/3179)
I added a note regarding the Web Modeler Ingress to the README file.